ansible/roles/purdue-rebuilderd-worker-vm/files/rebuilderd-worker-vm@.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=rebuilderd-worker-vm
+After=docker.service
+Requires=docker.service
+
+[Service]
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker stop rebuilderd-worker-vm-%i
+ExecStart=/usr/bin/docker run \
+--rm \
+--name rebuilderd-worker-vm-%i \
+--network=purdue-monitor \
+--device=/dev/kvm:/dev/kvm:rwm \
+--mount type=volume,source=vmrebuilderd,destination=/vm/ \
```

Comment:
I think this mount should be using the parameters from the unit to separate the instances of the VM image